Ontology/Asset/Equipment/HVACEquipment/AirHandlingUnit.json

```diff
@@ -261,29 +199,7 @@
         "en": "re-heating method"
       },
       "name": "reHeatingMethod",
-      "dtmi:dtdl:property:schema;2": {
-        "@type": "Enum",
-        "dtmi:dtdl:property:enumValues;2": [
-          {
-            "enumValue": "Electric",
-            "name": "Electric"
-          },
-          {
-            "enumValue": "Gas",
-            "name": "Gas"
-          },
-          {
-            "enumValue": "HotWater",
-            "name": "HotWater"
-          },
-          {
-            "enumValue": "None",
-            "name": "None"
-          }
-        ],
-        "valueSchema": "string"
-      },
-      "writable": true
+      "schema": "dtmi:digitaltwins:rec_3_3:asset:HVACHeatingMethod;1"
```

Comment:
A Property cannot have a schema that is an interface. This element could be changed to be a Component or the schema defined inline or in a reusable schema in the Interface's schemas element.